students/travis_nelson/lesson02/fizzBuzz.py

```diff
@@ -0,0 +1,38 @@
+
+def fizzBuzz1(fizzUntil=100):
```

Comment:
function names should follow snake_case convention -> `fizz_buzz1`, same with `fizzUntil` 

you can read pep8 style guide or use an online linter to catch violations - however, I usually recommend Pycharm as everything is configured out of the box and it will flag any violations as you write code.

Reply:
Understood! I'm looking at linters now (in VS Code), and will consider switching over to Pycharm. I'll definitely be more attentive to this in the future.